test/mocks/helpers/almaHelpers.js

```diff
@@ -11,7 +11,12 @@ function formatCustomerData() {
     };
 }
 
+function getMode() {
```

Comment:
If it's a constante for test maybe rename the function with getModeTest ?

Reply:
It's a mock for testing, in this case we want `getMode` to return "TEST" in order to simulate the use case in test mode :)